lib/JobLogs/JobLogs.js

```diff
@@ -54,6 +56,7 @@ JobLogsComponent.propTypes = {
   visibleColumns: PropTypes.arrayOf(PropTypes.string).isRequired,
   onSort: PropTypes.func.isRequired,
   onRowClick: PropTypes.func,
+  mclProps: PropTypes.object,
```

Comment:
Ordering issue. I think it should be in styleguide to place function props at the bottom.